source/optional_features/digital_wallets/index.html.md.erb

```diff
@@ -0,0 +1,68 @@
+---
+title: Digital wallets
+weight: 136
+---
+
+# Digital wallets
+
+You can enable [Google Pay](https://pay.google.com/intl/en_uk/about/) and [Apple Pay](https://www.apple.com/uk/apple-pay/) [external links] to take payments from your users.
+
+You can only enable Google Pay and Apple Pay if you have a [live Worldpay account](/switching_to_live/set_up_a_live_worldpay_account/#set-up-a-live-worldpay-account).
+
+## Enable Apple Pay
+
+Sign in to the [GOV.UK Pay admin tool](https://selfservice.payments.service.gov.uk/).
+
+1. Go to __Settings__ and select __Card types__.
+
+1. Select the __Digital wallet__ tab.
+
+1. Select __Enable__ next to Apple Pay.
+
+1. On the __Enable Apple Pay__ screen, select __Yes, turn on Apple Pay__.
+
+## Enable Google Pay
+
+To enable Google Pay, you must enter a unique Google Pay merchant ID into your GOV.UK Pay account.
+
+You get your Google Pay merchant ID from your Worldpay merchant admin interface.
+
+1. If required, speak to your Worldpay account manager to upgrade your Worldpay merchant admin interface, so that the interface has Google Pay functionality.
+
+1. Sign in to your [Worldpay merchant admin interface](https://secure.worldpay.com/sso/public/auth/login.html) [external link].
+
+1. Go to __Integration__ and select the __Pay with Google__ tab.
+
+1. Under __Generate unique merchant ID__, select __Generate__.
+
+1. Copy the generated merchant ID.
+
+Sign in to the [GOV.UK Pay admin tool](https://selfservice.payments.service.gov.uk/).
+
+1. Go to __Settings__ and select __Card types__.
+
+1. Select the __Digital wallet__ tab.
+
+1. Select __Enable__ next to Google Pay.
+
+1. On the __Enable Google Pay__ screen, select __Yes, turn on Google Pay__.
+
+1. Enter the generated merchant ID and select __Submit__.
+
+## 3D Secure
+
+If a digital wallet transaction requires 3D Secure authentication, and the user making that transaction has not enabled biometric authentication on their digital wallet, that user must complete 3D Secure authentication in the same way as a non-digital wallet transaction.
+
+Apple Pay has biometric authentication enabled by default, so Apple Pay transactions should not require 3D Secure authentication.
```

Comment:
Could we state that it’s the user that won’t be required to go through 3DS. So like 

```suggestion
Apple Pay has biometric authentication enabled by default, so during Apple Pay transactions users will not be required to go though the usual 3D Secure authentication steps.
```

Realise thats wordy and happy to be wrong?

Reply:
Thanks @jonheslop. I would tweak it slightly; we don't use terms like "the usual" because that can be subjective. Also, I would like to keep "should", as I imagine people can disable biometric auth within their apple pay wallet. How about: 
Apple Pay has biometric authentication enabled by default, so users should not have to go through 3D Secure authentication during Apple Pay transactions.


will not be required to go though the usual 3D Secure authentication steps.

Reply:
@jonathanglassman My understanding from https://support.apple.com/en-gb/HT201239 is that without biometrics the user would be prompted for a passcode and this still counts as a sufficiently secure transaction so there is no circumstances in which Apple Pay will trigger 3DS, so 
> will not 

is a more accurate wording.

Reply:
Thanks @gidsg will redraft

Reply:
@jonathanglassman ...may be more accurate to say *Apple Pay has 2 factor authentication* as this would encompass biometrics or a passcode.

Reply:
@gidsg So, should we in fact say:

```
A user making a digital wallet transaction must complete 3D secure authentication on that transaction if:

* that transaction needs 3D Secure authentication
* that user has not enabled two-factor authentication on their digital wallet

Apple Pay has two-factor authentication enabled by default, so users will not have to complete 3D Secure authentication during Apple Pay transactions.

Google Pay does not have two-factor authentication enabled by default.
```

This covers off both biometrics and other 2fa methods.

Reply:
Not sure _2 factor_ works because there isn’t really a first factor!

I think only using a passcode to authenticate only applies to Apple Pay Cash payments in store. On the Web, I think it always requires biometrics (Face ID or Touch ID) on on iPhone/iPad/Mac, or double-clicking the side button on an Apple Watch (I guess the fact that it’s attached to your wrist is proof enough that it’s you!).

I think saying that Apple Pay requires authentication and therefore 3D Secure won’t be used would be fine.

Reply:
So, is the following ok:

```
A user making a digital wallet transaction must complete 3D secure authentication on that transaction if:

* that transaction needs 3D Secure authentication
* that user has not enabled authentication on their digital wallet

Apple Pay enables authentication by default, so users do not have to complete 3D Secure authentication during Apple Pay transactions.

Google Pay does not enable authentication by default, so users may have to complete 3D Secure authentication during Google Pay transactions, depending on their configuration.
```

Reply:
Apple Pay doesn’t just enable authentication by default, it enforces it always. You can’t not use authentication with Apple Pay.

Also, rather than talking about the user having authentication enabled on their digital wallet, I think it might be simpler to talk about whether the user used authentication when making the payment (e.g. in the second bullet point, something like “the user did not securely authenticate when using their digital wallet to confirm the payment”).

Reply:
To clarify, "the user did not securely authenticate...the payment" means that there are other methods of secure authentication apart from 3D Secure, and the user did not use any of these non-3D Secure methods, e.g. biometrics

Reply:
@alexbishop1 @oswaldquek @tillwirth @katiebates @gidsg please take a look at the latest draft. I've changed the focus of the authentication section to be more about secure authentication overall, with a reference to 3D Secure as a subset of secure auth.